src/main/kotlin/lotto/domain/WinningLotto.kt

```diff
@@ -1,6 +1,14 @@
 package lotto.domain
 
 data class WinningLotto(
-    val winningNumbers: List<LottoNumber>,
+    val winningNumbers: LottoTicket,
     val bonusNumber: LottoNumber
-)
+) {
+    init {
+        checkWinningNumbersContainsBonusNumber()
+    }
+
+    private fun checkWinningNumbersContainsBonusNumber() {
+        if (winningNumbers.value.contains(bonusNumber)) throw IllegalArgumentException("보너스 숫자가 이미 당첨번호에 존재합니다.")
```

Comment:
`require()`는 값을 만족하지 않으면 `IllegalArgumentException`을 발생시킵니다.

```suggestion
        require(!winningNumbers.value.contains(bonusNumber)) { "보너스 숫자가 이미 당첨번호에 존재합니다." }
```